json/intermediate-large.json

```diff
@@ -207,5 +207,12 @@
       ],
       "exclude_node_ids": []
     }
-  ]
+  ],
+  "metadata": {
+    "title": "6 GB",
+    "subtitle": "Large",
+    "size": 6,
+    "text": "Intermediate",
+    "channels": 22
+  }
```

Comment:
I understand that the sources are INI files, and the output is JSON. So the metadata should go to the existing INI files.